attachment_image_resize/models/ir_attachment.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2024 Quartile
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+IMAGE_TYPES = ["image/png", "image/jpeg", "image/bmp", "image/tiff"]
+
+
+class IrAttachment(models.Model):
+    _inherit = "ir.attachment"
+
+    resize_done = fields.Boolean(
+        help="Indicates whether the resizing process has been run on this record. "
+        "Once selected, the record will be excluded from "
+        "the _cron_resize_attachment_image() target."
+    )
+
+    def _postprocess_contents(self, values):
+        self = self.with_context(
+            resize_target_model=self.res_model or values.get("res_model")
```

Comment:
Shouldn't this rather be in this order?
```suggestion
            resize_target_model=values.get("res_model") or self.res_model
```